### DISCUSSION TITLE
The latitude and longitude are not being applied when editing a photo in the Immich Android app.

### DISCUSSION THREAD

**Discussion by fredsco** [immich-app/immich#11411] [Q&A]
### The bug

Upon pressing the information button within a picture and editing the location, the lat/lon does not get applied and stays at 0. I have confirmed the lat/lon values are correct. 

### The OS that Immich Server is running on

Android

### Version of Immich Server

1.109.2

### Version of Immich Mobile App

1.110 build.151

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

```bash
1. Open an image
2. Press info button on the right top
3. modify/add the lat/lon
...
```

**Comment by bo0tzz** [comment]
Please post the compose file and such, they are actually relevant. Are you using an external library that's mounted readonly? If so, metadata edits won't work.

**Comment by VibrantTreeStump** [comment]
I've had this exact same issue. I found that you need to do this:
<img src="https://github.com/user-attachments/assets/d908c8d0-8ad1-4093-973e-5bf3d07ea59d" width=20%>

If you tap on update without first confirming the number on your keyboard, the change won't be recognized.
